packages/kotti-ui/source/kotti-navbar/types.ts

```diff
@@ -1,39 +1,52 @@
-import { Yoco } from '@3yourmind/yoco'
+import { yocoIconSchema } from '@3yourmind/yoco'
+import { z } from 'zod'
 
 import { SpecifyRequiredProps } from '../types/utilities'
 
 export namespace KottiNavbar {
-	export type QuickLink = {
-		link: string
-		title: string
-	}
-
-	export type SectionLink = {
-		icon: Yoco.Icon
-		isActive: boolean
-		title: string
-		to: unknown
-	}
-
-	export type Section = {
-		title: string | null
-		links: SectionLink[]
-	}
+	export const notificationSchema = z.object({
+		count: z.number(),
+		link: z.string(),
+		title: z.string(),
+	})
+	export type Notification = z.infer<typeof notificationSchema>
+
+	export const quickLinkSchema = z.object({
+		link: z.string(),
+		title: z.string(),
+	})
+	export type QuickLink = z.infer<typeof quickLinkSchema>
+
+	export const sectionLinkSchema = z.object({
+		icon: yocoIconSchema,
+		isActive: z.boolean(),
+		title: z.string(),
+		to: z.unknown().optional(),
```

Comment:
All the usages in the user-app rely on a `to` attribute on the section
> but it's internally used on the click handler that we eventually pass to `KtNavbar`
> so there is no breaking usage

HOWEVER, exposing the `to` will confuse users who inspect the type not the actual implementation

SO:
1) remove the `to` 
2) introduce the `link` prop which is a `string | undefined` that we bind to `href` in `NavbarMenu.vue`

The user either uses the `link` and relies on our `href` implementation, or they have whatever internal API that they like and exclusively use the click handler

This comment can be resolved with this:
https://github.com/3YOURMIND/kotti/pull/504/commits/257ea964d015d82203c72e3631fc4b40b316e228#r713940508